types/react-native/index.d.ts

```diff
@@ -9170,6 +9170,7 @@ export interface I18nManagerStatic {
     getConstants: () => {
         isRTL: boolean;
         doLeftAndRightSwapInRTL: boolean;
+        localeIdentifier?: string;
```

Comment:
Given that this is a [maybe type](https://flow.org/en/docs/types/maybe), we should make it:
```suggestion
        localeIdentifier?: string | null;
```
:)